webapi/scripts/expected-client-test-data.txt

```diff
@@ -29,38 +29,11 @@ test-data/admin/lists/delete-list-response.json
 test-data/admin/lists/get-anything-project-lists-response.json
 test-data/admin/lists/get-image-project-lists-response.json
 test-data/admin/lists/get-list-info-response.json
-test-data/admin/lists/get-list-node-info-response.json
```

Comment:
This test-data-file is now missing in js-lib. What was the reason to delete the file?

Reply:
And what will be the equivalent test-data for all the deleted "feature_toggle" lists?

Reply:
Test data are generated in relation to E2E tests on DSP-API side, if the test was deleted, the test data file needs to be deleted too. In this case, the test was redundant. In the case of lists - whole new implementation is gone, so the related tests and test-data are.